CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# [1.4.0](https://github.com/algolia/search-insights.js/compare/v1.3.1...v1.4.0) (2020-02-17)
+
+
+### Bug Fixes
+
+* **gtm:** add Algolia Agent after `init` ([9640dc2](https://github.com/algolia/search-insights.js/commit/9640dc2677c931cdfd1e278834c54fe169381ba6))
+* **gtm:** forward `eventName` to all events ([1209301](https://github.com/algolia/search-insights.js/commit/12093016e424e0a0fce3fe803cc5388d5a905219))
+* **types:** remove optional types on main methods ([#192](https://github.com/algolia/search-insights.js/issues/192)) ([e551d10](https://github.com/algolia/search-insights.js/commit/e551d106464edbcc5304047d398a7b0df48c0823))
+
+
+### Features
+
+* **sendEvent:** return the value of the request function ([#200](https://github.com/algolia/search-insights.js/issues/200)) ([56b4ffd](https://github.com/algolia/search-insights.js/commit/56b4ffd0e878a02767f9553ef9dce26bf6dc6d26)), closes [#199](https://github.com/algolia/search-insights.js/issues/199)
```

Comment:
I think this was detected as a major version, since I have:

> The existing request functions return undefined, so this isn't a breaking change, but this allows you to have a custom requester you can wait on (e.g. fetch).


in the commit description, which includes "breaking change", but not at the beginning of a line. @eunjae-lee is it possible that shipjs miscategorises commits like these?

Anyway, it wasn't an issue, since I could change the version number.

Reply:
@Haroenv yes, it's on the list: https://github.com/algolia/shipjs/issues/634
Thanks for reminding me!